compiler/ballerina-lang/src/test/java/org/wso2/ballerinalang/compiler/BLangNodeTransformerTest.java

```diff
@@ -280,16 +246,12 @@ private static JsonObject fileContentAsObject(String filePath) {
         return JSON_PARSER.parse(contentAsString).getAsJsonObject();
     }
 
-    /**
-     * Represents an empty print stream to avoid writing to the standard print stream.
-     */
-    private static class EmptyPrintStream extends PrintStream {
-        public EmptyPrintStream() throws UnsupportedEncodingException {
-            super(new OutputStream() {
-                @Override
-                public void write(int b) {
-                }
-            }, true, "UTF-8");
-        }
+    private static Project loadProject(String sourceFilePath) {
+        Path sourcePath = Paths.get(sourceFilePath);
+        String sourceFileName = sourcePath.getFileName().toString();
+        Path sourceRoot = RES_DIR.resolve(sourcePath.getParent());
+        Path projectPath = Paths.get(sourceRoot.toString(), sourceFileName);
+        BuildOptionsBuilder buildOptionsBuilder = new BuildOptionsBuilder();
+        return ProjectLoader.loadProject(projectPath, buildOptionsBuilder.build());
```

Comment:
We can use `ProjectLoader.loadProject(projectPath)`. It will set the default build options.